VoiceTransformer/VoiceTransformer.ipynb

```diff
@@ -38,7 +38,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.rcParams[\"figure.figsize\"] = (14,4)"
+    "plt.rcParams[\"figure.figsize\"] = (14,4)\n",
+    "plt.rcParams['image.cmap'] = 'tab10'"
```

Comment:
to make sure everyone uses the same default colormap (otherwise it is hard to refer to the colors in plots consistently)